Sources/KSCrashRecording/KSCrashC.c

```diff
@@ -176,7 +176,7 @@ KSCrashMonitorType kscrash_install(const char* appName, const char* const instal
 
     snprintf(path, sizeof(path), "%s/Data", installPath);
     ksfu_makePath(path);
```

Comment:
nit: The path is ensured here so you can remove the dir creation from `ksmemory_initialize`